regression/run-all.sh

```diff
@@ -11,6 +11,7 @@
 
 chmod a+x cbmc/run-all.sh
 chmod a+x esbmc/run-all.sh
+chmod a+x settling-time_det/run-all.sh
```

Comment:
All regression tests should be located under `cbmc` or `esbmc` subdirectory.

Reply:
DONE!